tests/RedirectedURLHandlerTest.php

```diff
@@ -41,25 +42,46 @@ public function testHandleURLRedirectionFromBase() {
 	public function testHandleURLRedirectionWithQueryString() {
 		$response = $this->get('query-test-with-query-string?foo=bar');
 		$expected = $this->objFromFixture('RedirectedURL', 'redirect-with-query');
-		
+
 		$this->assertEquals(301, $response->getStatusCode());
 		$this->assertEquals(
 			Director::absoluteURL($expected->To),
 			$response->getHeader('Location')
 		);
 	}
 
-	public function testArrayToLowercase() {
-		$array = array('Foo' => 'bar', 'baz' => 'QUX');
-		
-		$cont = new RedirectedURLHandler();
+	public function testHandleMixedCaseURLRedirection() {
+		$response = $this->get('Query-test-with-mixed-case-url');
+		$expected = $this->objFromFixture('RedirectedURL', 'redirect-with-mixed-case-url');
 
-		$arrayToLowercaseMethod = new ReflectionMethod('RedirectedURLHandler', 'arrayToLowercase');
-		$arrayToLowercaseMethod->setAccessible(true);
-		
+		$this->assertEquals(301, $response->getStatusCode());
+		$this->assertEquals(
+			Director::absoluteURL($expected->To),
+			$response->getHeader('Location')
+		);
+	}
+
+	public function testHandleURLRedirectionWithMixedCaseQuery() {
+		$response = $this->get('query-test-with-mixed-case-query-string?Foo=bar');
```

Comment:
Like you've done in `testHandleURLRedirectionWithMixedCaseQueryAndCaseInsensitiveMatching` to set the config flag to true, I think it'd be worth doing it here also to set it to false - this will ensure that the global/project state won't affect your test results (e.g. if a user sets it to false in their mysite configuration)